website/redirects.json

```diff
@@ -171,11 +169,11 @@
 {"source": "ingestion/overview.html", "target": "index.html"}
 {"source": "ingestion/realtime-ingestion.html", "target": "index.html"}
 {"source": "ingestion/reports.html", "target": "tasks.html#reports"}
-{"source": "ingestion/schema-changes.html", "target": "data-management.html#schema-changes"}
+{"source": "ingestion/schema-changes.html", "target": "faq.html#how-can-i-reindex-existing-data-in-druid-with-schema-changes"}
```

Comment:
I think the target should be `design/segments.html#segments-with-different-schemas`.

The source of `schema-changes.html` was probably some version of https://apache.googlesource.com/druid/+/refs/heads/0.13.0-incubating/docs/content/ingestion/schema-changes.md.

The same content is in https://github.com/apache/druid/blob/master/docs/design/segments.md

Reply:
sgtm, changed